configure

```diff
@@ -6,17 +6,18 @@
 _=[ 'exec' '/bin/sh' '-c' '''
 which python2.7 >/dev/null && exec python2.7 "$0" "$@"
 which python2 >/dev/null && exec python2 "$0" "$@"
+which python3 >/dev/null && exec python3 "$0" "$@"
 exec python "$0" "$@"
 ''' "$0" "$@"
 ]
 del _
 
 import sys
 from distutils.spawn import find_executable as which
-if sys.version_info[0] != 2 or sys.version_info[1] not in (6, 7):
-  sys.stderr.write('Please use either Python 2.6 or 2.7')
+if sys.version_info[0] not in (2, 3) or sys.version_info[1] != 7:
+  sys.stderr.write('Please use either Python 2.7 or 3.7')
```

Comment:
Not 3.6? That will be the default in Travis from 16th April 2019: https://changelog.travis-ci.com/upcoming-python-default-version-update-96873

Reply:
It was just easier to check... I wanted a quick audit of our CI...

tl;dr It won't land like this as this will fail multiple CI jobs (and anyway there's other PR that need to land for even 3.7 to work)

Reply:
`if sys.version_info[:2] not in ((2, 7), (3, 6), (3, 7)):`

Reply:
Fixed